core/templates/dev/head/pages/dashboard/Dashboard.js

```diff
@@ -215,3 +209,23 @@ oppia.controller('Dashboard', [
     );
   }
 ]);
+
+oppia.directive('explorationTitle', function($compile) {
```

Comment:
Rename this to something more meaningful ? Also, please move this directive to https://github.com/oppia/oppia/tree/develop/core/templates/dev/head/domain/dashboard (See ScrollOnClickDirective.js)